### ISSUE TITLE
Tune the email templates during peak traffic

### ISSUE THREAD

**Issue by pixdemo** [KeypDemoCorp/awesome-api-backend#284]
Performance is noticeably degraded under load and should be improved.

Accessibility should be considered when implementing this.

This is a good opportunity to clean up some of the surrounding code.

**Comment by pixdemo** [comment]
Do we have a sense of how many users this affects?
